demo/pom.xml

```diff
@@ -55,11 +78,75 @@
                 <version>1.1.1</version>
                 <configuration>
                     <mainClass>com.dooapp.fxform.Demo</mainClass>
+                    <commandlineArgs>
+                        --add-exports=javafx.graphics/com.sun.javafx.scene.control.skin=ALL-UNNAMED
+                    </commandlineArgs>
                 </configuration>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+
+            </plugin>
+            <plugin>
+                <groupId>org.openjfx</groupId>
+                <artifactId>javafx-maven-plugin</artifactId>
+                <version>0.0.8</version>
+                <configuration>
+                    <mainClass>com.dooapp.fxform.Demo</mainClass>
+                    <options>
+                        <!--                        <option>-verbose</option>-->
+                        <option>-Dprism.maxvram=2G</option>
```

Comment:
Why is this option required ?

Reply:
I am sorry, 
That is a mistake. I will be remove in the next commit